src/Apps/Order/Dialogs.tsx

```diff
@@ -0,0 +1,157 @@
+import { Link } from "@artsy/palette"
+import { ModalDialog, ModalDialogProps } from "Components/Modal/ModalDialog"
+import React from "react"
+import { Container, Subscribe } from "unstated"
+
+interface DialogState {
+  props: ModalDialogProps
+  forceCloseDialog: () => Promise<void>
+}
+
+export class DialogContainer extends Container<DialogState> {
+  state: DialogState = {
+    props: {
+      show: false,
+      heading: null,
+      detail: null,
+      primaryCta: {
+        text: "",
+        action: () => void 0,
+      },
```

Comment:
Any particular reason not to initialize `secondaryCta` and `onClose` here?

Reply:
`secondaryCta` is optional, and this initial value is not really used anyway* so there's no need to specify it. 

*It's only used so that we don't need to have logic for handling `null` or `undefined` when rendering the  actual `ModalDialog`